uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/CheatSheetsTest.java

```diff
@@ -0,0 +1,197 @@
+import org.eclipse.reddeer.uiforms.impl.formtext.DefaultFormText;
+import org.eclipse.reddeer.uiforms.impl.hyperlink.DefaultHyperlink;
+import org.eclipse.reddeer.uiforms.impl.section.DefaultSection;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.eclipse.reddeer.workbench.impl.view.WorkbenchView;
+import org.eclipse.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.jboss.tools.fuse.reddeer.condition.HyperLinkHasText;
+import org.jboss.tools.fuse.reddeer.preference.InstalledJREs;
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizard;
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardFirstPage;
+import org.jboss.tools.fuse.reddeer.wizard.NewFuseIntegrationProjectWizardRuntimePage;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author fpospisi, djelinek
+ */
+@RunWith(RedDeerSuite.class)
+public class CheatSheetsTest {
+
+	public static final String TITLE = "Create a Fuse Integration Project";
+
+	public static final String INTRODUCTION = "This cheat sheet walks you through the steps "
+			+ "to create a new Fuse Integration project based on Spring Boot.";
+
+	public static final String PERSPECTIVE = "Select Window > Open Perspective > Fuse Integration "
+			+ "in the menu bar at the top of the workbench. This step changes the perspective "
+			+ "to set up the Eclipse workbench for development of your Apache Camel/Red Hat Fuse " + "routes.";
+
+	public static String WIZARD = "To create your new project you now need to "
+			+ "open the New Integration Project wizard. You can do that by selecting "
+			+ "File > New > Fuse Integration Project from the top menu bar.Once it "
+			+ "opened, you can specify the location your project should live in. On "
+			+ "the next page, you can select specify the target environment for deploying "
+			+ "your new project and choose a specific Camel version. On the last page, you "
+			+ "can select from a list of available templates. Please select the template "
+			+ "Simple log using Spring Boot - Spring DSL from the list.Click on Finish to "
+			+ "finally create your new project.";
+
+	@BeforeClass
+	public static void setWorkspace() {
+	}
+
+	@AfterClass
+	public static void cleanWorkspace() {
+		new CleanWorkspaceRequirement().fulfill();
+	}
+
+	/**
+	 * <p>
+	 * Fuse Integration Project cheat sheet test
+	 * </p>
+	 * <b>Steps</b>
+	 * <ol>
+	 * <li>Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse integration project</li>
+	 * <li>Check if texts in all sections are actual.</li>
+	 * <li>Activate available HyperLinks and wait for action.</li>
+	 * <li>Check according actions.</li>
+	 * <li>Go through New Project Wizard.</li>
+	 * </ol>
+	 */
+	@Test
+	public void testCheatSheets() {
+
+		boolean hasJava8 = hasJava8Available();
+
+		/*
+		 * Open Help -> Cheat Sheets -> Red Hat Fuse -> Create a Fuse...
```

Comment:
It would be nicer to have new e.g. `FuseCheatSheetsDialog` class which could contains whole this logic, it will simplify test code